.github/workflows/tagbot.yml

```
@@ -0,0 +1,31 @@
+name: Tagbot
+on: [pull_request]
+
+permissions:
+  pull-requests: write
```

```suggestion
  contents: read
  pull-requests: write
```
don't you need read permissions as well?

---

I've tested this workflow separately in a junk repo and that was enough to specify. I'm just having some troubles getting the correct git checkout here when i have remote branches (working on it)